app/controllers/sign_up/registrations_controller.rb

```diff
@@ -15,6 +15,7 @@ def new
         attempts_tracker: irs_attempts_api_tracker,
       )
       analytics.user_registration_enter_email_visit
+      @a_b_test_bucket = AbTests::SIGN_IN.bucket(request.ip)
```

Comment:
I would consider calling this `@sign_in_form_bucket` so it's clear what test is being bucketed
```suggestion
      @sign_in_form_bucket = AbTests::SIGN_IN.bucket(request.ip)
```

Reply:
Renamed in 2f3943a